### PR TITLE
java CDK: try to fix flaky LsnCommitTest

### PR DESCRIPTION
**Pull request by postamar**
This is still flaky. Now using `pg_current_wal_insert_lsn` instead of `pg_current_wal_lsn` in the hopes that this test will finally behave deterministically.

### CODE REVIEW

File: airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/PostgresUtils.java [hunk 1]
```diff
@@ -18,12 +18,21 @@ public class PostgresUtils {
 
   public static PgLsn getLsn(final JdbcDatabase database) throws SQLException {
     // pg version >= 10. For versions < 10 use query select * from pg_current_xlog_location()
-    final List<JsonNode> jsonNodes = database
-        .bufferedResultSetQuery(conn -> conn.createStatement().executeQuery("select * from pg_current_wal_lsn()"),
-            resultSet -> JdbcUtils.getDefaultSourceOperations().rowToJson(resultSet));
+    return getLsnImpl(database, "pg_current_wal_lsn");
+  }
+
+  public static PgLsn getInsertLsn(final JdbcDatabase database) throws SQLException {
```
Comment (rodireich) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/PostgresUtils.java:24]: Do we have any test with pgVer < 10

Comment (postamar) [reply airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/PostgresUtils.java:24]: I don't think so, but I can easily (and gladly) add that. Which versions of pg should we support?

**Comment by postamar** [comment]
Adding these extra cases for more postgres versions triggered the flakiness quite reliably. I've come to the conclusion that we simply can't control what goes into the WAL (postgres just does whatever it likes, as it should) and so I've given up trying to test for a lack of progress.

**Comment by postamar** [comment]
I'm going to undo the debezium short-circuiting entirely, doing so does not slow down the tests significantly.

**Pull request closed without merging**